### ISSUE TITLE
Marking article text advances to the next article

### ISSUE THREAD

**Issue by dsale** [wallabag/wallabag#1309]
In Firefox, with all browsing extensions disabled, marking an article's text causes Wallabag to suddenly advance to/load the next article.

**Comment by tcitworld** [comment]
I'm not sure what you mean by marking the article's text. I don't manage to reproduce it.

**Comment by dsale** [comment]
I mark words in the text with my mouse in order to highlight it, so I can copy/paste. Happens in both Firefox and Chrome.

**Comment by tcitworld** [comment]
Restarted with both clean versions of Firefox and Chrome. Nothing.

Can you post your versions of your browser and operating system ? Do you use any software to handle your mouse that could emulate keyboard while selecting text ? Are you using the main button of your mouse to select text or a dedicated button as some mouses have.

**Comment by dsale** [comment]
I just noticed something. If I mark to the right, it goes to the next article. If I mark to the left, it goes to the previous one. Marking the article title does nothing. Just the article body.

Using the latest version of Firefox / Chrome & Windows 10.

**Comment by tcitworld** [comment]
> Marking the article title does nothing. Just the article body.

The detection of keyboard keys is just on the body part, yup.

I'll reboot to Windows 10 later to check, but I never experienced it even on that platform. I think your mouse driver messes with things. Will try looking for a solution.

**Comment by dsale** [comment]
Might be the mouse driver. I dunno. It's a fairly new Logitech & I haven't experienced any issues with it before this new Wallabag install.

**Comment by tcitworld** [comment]
I must find some place where you could confirm this apart from wallabag. Then, I'll see if we can stop the mouse events being detected in wallabag.

**Comment by dsale** [comment]
The only thing I notice with the Wallabag installation I've installed now and the last installation I had going is the addition of the arrows at the bottom right. I don't know if they have anything to do with it, it's just an observation.

**Comment by tcitworld** [comment]
I should have started my explanation this way :
It's totally this that does it. In addition to the arrows, there are keyboard shortcuts that takes you, for instance, to the next or previous article. One of these shortcuts are keyboard arrows.

**Comment by dsale** [comment]
Ok gotcha.

**Comment by dsale** [comment]
Hah, I should've checked this earlier, but it's happening on my tablet, with no mouse. I just swipe the article text, and it goes to another article.

Could this be PHP rekated? The version I'm using is 5.4.

**Comment by tcitworld** [comment]
Nope, that's a intended functionality. ;-) 

PHP should have nothing to do with this. It's JavaScript and client related.

**Comment by KeiOtori** [comment]
Found this issue too.

The problem is: Firefox imitates lift-right swipe with mouse. If page supports this functionality (get swipe -> do something like "go to the next page"), at the same time we select something on page and go next.

In my wallabag i have two bits of this:
- *index.php* (loaded on client side!), line 576 ~ 579:
```    
        // swipe to right or left on mobile to navigate
        $('article').on("swiperight", function(){
            goLeft('?view=view&id=');
        });
        $('article').on("swipeleft", function(){
            goRight('?view=view&id=370');
        });
```
If firefox get swipe from mouse, it runs the functions in keyboard.js:
```
goLeft(..);
goRight(..);
```
My (very) quick fix for this issue: comment the corresponiding lines (like in *index.php* over) in *wallabag/themes/default/view.twig* on your server (and clear cache of the pages).
But, after that, swipe will not work on touch devices.

**Comment by tcitworld** [comment]
Oh, right. It's the swipe events which are triggered. Got it, thanks.

**Comment by bpavot** [comment]
Same swipe event detected on both browser:
* Chromium Version 45.0.2454.101 Ubuntu 14.04 (64-bit)
* Firefox 41.0.2  Ubuntu 14.04 (64-bit)

**Comment by FM33** [comment]
I can reproduce it on framabag. It also matches the issue I have on android : If I don't scroll strictly vertically, then previous/next article is loaded. Very annoying.
Firefox 42, Ubuntu 14.04 and Firefox 41, Android 4.4

**Comment by nicosomb** [comment]
It affects only wallabag v1. Won't be fixed, we work only on wallabag v2.

**Comment by nicosomb** [comment]
We'll released a 1.9.2 \o/
